### DISCUSSION TITLE
[Projects Beta] Linked repositories

### DISCUSSION THREAD

**Discussion by sebdeleze** [community/community#8884] [Projects and Issues]
It would be great if we can associate one or more repositories in the project settings. It will reduce the choices when a draft is converting to an issue.

**Comment by ravanar-sk** [comment]
Yes, that's the first thing I tried to find out about the new Projects.

**Comment by biwin** [comment]
Is the current design 'Not to link repos' and convert the draft to issue 'in whatever repo' ? 
If yes, give a choice to select 'prefered repos' or at least keep the last used repo (when converting) in the suggestions?
Searching each time makes the process quite boring (repeating the same things over and over) :unamused:

**Comment by StephenWithPH** [comment]
Also many requests for this feature here: https://github.com/github/feedback/discussions/5867

**Comment by matthewferry** [comment]
Want this too! Right now it's tedious to make sure every issue created in a repo sets the project in the sidebar or go to the project and type/search for the new issue. Would love if maybe there was a preset workflow called "Issue created in repo" that:

When: `issue, pull request` is created in `[org/repos]`
Add to: this project 
Set: `values`

**Comment by ivanaguilario** [comment]
Yes, this to me seems like an essential feature!
